sanic/server/websockets/impl.py

```diff
@@ -840,3 +845,8 @@ def connection_lost(self, exc):
         self.abort_pings()
         if self.connection_lost_waiter:
             self.connection_lost_waiter.set_result(None)
+
+    async def __aiter__(self):
+        with suppress(ConnectionClosedOK):
+            while True:
+                yield await self.recv()
```

Comment:
```python
try:
    something_bad()
except Exception:
    ...
```
This is almost always going to be faster than `suppress`.

Reply:
Okay, change it back to try except pair

Reply:
Benchmarked, yes `suppress` is slower by about 220 ns. Most definitely negligible compared to the opening of a websocket, or to even just the recv call, but still eight times the time it takes to make a simple function call with only try-except within that function and nothing else.